src/org/exist/collections/CollectionConfigurationManager.java

```diff
@@ -117,7 +117,7 @@ public void addConfiguration(Txn txn, final DBBroker broker, Collection collecti
 
             XmldbURI configurationDocumentName = null;
             // Replaces the current configuration file if there is one
-            final CollectionConfiguration conf = getConfiguration(broker, collection);
+            final CollectionConfiguration conf = getConfiguration(broker, confCol);
```

Comment:
without this change... what went wrong?

Reply:
hmm good question. I think that was a mistake. I will correct